mlflow/pytorch/torchserve/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name="torchservedeployment",
```

Comment:
```suggestion
    name="mlflow-torchserve",
```

Maybe name the deploy plugin something that suggests a connection to MLflow?